README.md

```diff
@@ -18,6 +18,12 @@ libraryDependencies += "org.broadinstitute.dsp" % "helm-scala-sdk_2.13" % "0.0.1
 ```
 
 - Set JVM parameter `-Djna.library.path=<path-to-generated-shared-go-library>`
+- Be sure to run the following helm commands before using this library
```

Comment:
shall we move this to `HelmManualTest.scala`? cuz I think the library itself isn't intended to support galaxy alone

Reply:
Hmm... I'll specify that these cmds are galaxy specific, but I think its worth mentioning that you will need to update helm directly for whatever charts you want it to interact with since its a bit confusing that you need to do something on the host machine for the binary to work